### ISSUE TITLE
KCom2 - Entetr "Tracking code" popup message always appearing when Order status changes

### ISSUE THREAD

**Issue by stephen-fueldesign** [Kademi/kademi-dev#6242]
I am always getting the popup message that asks something like "enter you tracking code if you have one"... even if I have one in place already.

**Comment by bradmac** [comment]
Yep, that is as intended.

**Comment by stephen-fueldesign** [comment]
Are you sure? Put another way, the popup does not need to appear if I have a tracking code already. What's the point? If you really must have the popup, even though a popup is present, can it at least auto-populate with the tracking code that's ready been saved and entered?

**Comment by stephen-fueldesign** [comment]
Are you sure? Put another way, the popup does not need to appear if I have a tracking code already. What's the point? If you really must have the popup, even though a popup is present, can it at least auto-populate with the tracking code that's ready been saved and entered?

**Comment by bradmac** [comment]
> can it at least auto-populate with the tracking code

yep, that would be good. But i think people dont actually change tracking codes very often, so this i wouldnt rate this as a high priority update.
